ma/maInput.cc

```diff
@@ -26,6 +26,45 @@ Input::~Input()
     delete solutionTransfer;
 }
 
+Input::Input(Input&& in)
+{
+  mesh = in.mesh;
+  sizeField = std::move(in.sizeField);
```

Comment:
move doesn't actually move anything...just casts to r-value reference, so this isn't doing what you expect. This should just be `std::swap`. Which will put the implicitly initialized `nullptr` to the old input.